src/shared/components/case-editor/services/cases.service.ts

```diff
@@ -297,13 +297,16 @@ export class CasesService {
   }
 
   private processTasksOnSuccess(caseData: any, eventData: any): void {
-    this.workAllocationService.completeAppropriateTask(caseData.id, eventData.id)
-      .subscribe(() => {
-        // Success. Do nothing.
-      }, error => {
-        // Show an appropriate warning about something that went wrong.
-        console.warn('Could not process tasks for this case event', error);
-      });
+    // This is used a feature toggle to 
+    // control the work allocation
+    if(this.appConfig.getWorkAllocationApiUrl()) {
+        this.workAllocationService.completeAppropriateTask(caseData.id, eventData.id)
+          .subscribe(() => {
+            // Success. Do nothing.
+          }, error => {
```

Comment:
Do we need the comments?

Reply:
@phillip-whitaker Agree but, I want the comment to make it explicit to the developer that config needs a null which is used a feature toggle.